docs/basics/101-105-install.rst

```diff
@@ -12,10 +12,10 @@ But we've been told that with DataLad we could very easily get vast amounts of d
 computer. Rumor has it that this would be only a single command in the terminal!
 Therefore, everyone in today's lecture excitedly awaits today's topic: Installing datasets.
 
-"With DataLad, users can *install* existing
-DataLad datasets from paths, URLs, or open-data collections." our lecturer begins.
+"With DataLad, users can get *clones* of existing DataLad datasets from paths, URLs, or
```

Comment:
maybe "install clones", instead of "get clones" to make the connection of the terms at first use